src/v0/destinations/braze/transform.js

```diff
@@ -326,7 +326,7 @@ function getPurchaseObjs(message) {
         }
         let purchaseObj = addMandatoryPurchaseProperties(
           productId,
-          price,
+          parseFloat(price),
```

Comment:
is this scenario covered in test cases ?

Reply:
Added tests